tests/json-fixtures/test_virtual_machine.py

```diff
@@ -57,13 +57,22 @@
 
 ROOT_PROJECT_DIR = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
 
-BASE_FIXTURE_PATH = os.path.join(
+# up-to-Constantinople snapshot legacy VM tests
+LEGACY_CONSTANTINOPLE_SNAPSHOT_TESTS = os.path.join(
     ROOT_PROJECT_DIR,
     "fixtures",
     "LegacyTests",
     "Constantinople",
     "VMTests",
 )
+# TODO: Look into adding these tests? Though they seem to be covered by the state tests
```

Comment:
Are these comments still relevant here?

Reply:
Yeah, it was a general observation I don't think is super important but wanted to keep the TODO there